### DISCUSSION TITLE
Is there a maximum number of connections that can be proxied

### DISCUSSION THREAD

**Discussion by wlarsong** [tg123/sshpiper#229] [Q&A]
Hello I have setup SSH piper to proxy connections between different servers. I have over 2,500 connections and I am finding once I reach 512 connections the SSH piper drops connections.

If I bypass SSH-Piper and all the connections go to one server the server can handle all 2500 without drops. As soon as I add ssh-piper I hit a 512 limit. 

Is SSH-piper using a config or line of code to control max connections to one server?

If I use SSH-piper and go to another server with less connections, everything is OK.

```
user1 *2500 ------------------>ssh-piper ------> Box 1 (Not working Capped at 512)
user2 *5 ----------------------^      |
                                      |-------->Box 2 (  Working all 5 can log in)
```

**Comment by tg123** [comment]
did not test so many conns to single piper, but thanks for the input.

there is no cap for sshpiper, did you see any err when accepting connection?
